src/util/simplify_expr_int.cpp

```diff
@@ -1670,17 +1670,18 @@ bool simplify_exprt::simplify_inequality_not_constant(exprt &expr)
   return true;
 }
 
-/// \par parameters: an inequality with a constant on the RHS
-bool simplify_exprt::simplify_inequality_constant(exprt &expr)
+/// \par parameters: an inequality where the RHS is a constant
```

Comment:
Since you are updating the comment: Could you please replace the `\par parameters:` by `\param expr:`?

Reply:
done